tor_ocr/main.py

```diff
@@ -76,8 +84,36 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+def ocr_space_url(url, overlay=False, api_key=__OCR_API_KEY__):
```

Comment:
The naming of this doesn't quite feel right to me. `ocr_space_url` seems more like we're returning a result url to check back on the OCR-ing task for a given image, but in reality we're just wrapping the entire call to the API and returning the response JSON directly. It returns a full set of data, not just a URL.

What about naming it something like `decode_image_from_url`? I'm sure there's a better name out there, but just an idea.

Reply:
`ocr_space_api` maybe? I'll use yours for now.